packages/uui-popover-container/lib/uui-popover-container.element.ts

```diff
@@ -79,10 +81,32 @@ export class UUIPopoverContainerElement extends LitElement {
     this.addEventListener('beforetoggle', this.#onBeforeToggle);
   }
 
+  protected override firstUpdated(_changedProperties: PropertyValues): void {
+    super.firstUpdated(_changedProperties);
+
+    this.#sizeObserver = new ResizeObserver(entries => {
```

Comment:
Does firstUpdated gets triggerede again when a component has been shortly out of the DOM?

I'm thinking that the sizeObserver gets disconnected on `disconnectedCallback`, and then I would be afraid that it never sets up the sizeObserver again.